multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/compose/elements/MarkdownText.kt

```diff
@@ -80,12 +89,10 @@ fun MarkdownText(
     }
 
     // forward the `onTextLayout` to `extended-spans` if provided
-    val onTextLayout: (TextLayoutResult, Color?) -> Unit = if (extendedSpans != null) {
+    val onTextLayout: ((TextLayoutResult, Color?) -> Unit)? = extendedSpans?.let {
```

Comment:
please revert to the `if/else` as I do think it's easier to read in this context.